platforms/android/src/jp/wizcorp/phonegap/plugin/WizAssets/WizAssetsPlugin.java

```diff
@@ -137,6 +135,7 @@ private void setBlockSize() {
             blockSizeLong = 16384;
         }
         blockSize = (int)blockSizeLong;
```

Comment:
Please return directly the value here (`(int)blockSizeLong`), otherwise it stays a setter.